internal/venice-client-common/src/main/java/com/linkedin/venice/utils/VeniceProperties.java

```diff
@@ -107,6 +107,50 @@ public VeniceProperties clipAndFilterNamespace(String nameSpace) {
     return builder.build();
   }
 
+  /**
+   * This method looks for all properties that begin with either of the given
+   * namespaces. Once those properties are identified, it removes the matching
+   * namespace and returns the properties.
+   *
+   * This enables support for dynamic Kafka configurations. All Kafka
+   * properties can follow a convention of namespace, and the properties
+   * are extracted and supplied to the Kafka Producer/Consumer.
+   *
+   * @param nameSpace1 The first namespace to look for.
```

Comment:
Is the only difference between clipAndFilterNamespace is that this takes two namespaces?

Could we just do an end all and make this take a list?

Reply:
Yeah they are same but with diff args. 

>Could we just do an end all and make this take a list?

I like this idea. Will update to use `Set`

Reply:
Done